webapp/channels/src/actions/command.ts

```diff
@@ -194,14 +194,22 @@ export function executeCommand(message: string, args: CommandArgs): ActionFunc {
                     case AppCallResponseTypes.NAVIGATE:
                         return {data: true};
                     default:
-                        return createErrorMessage(localizeAndFormatMessage(
-                            t('apps.error.responses.unknown_type'),
-                            'App response type not supported. Response type: {type}.',
-                            {type: callResp.type},
+                        return createErrorMessage(intlShim.formatMessage(
```

Comment:
As a side note, I was a bit confused by this `intlShim` at first, but I actually really like that idea for handling difficult cases like this when we do commit to removing `localizeMessage`

Reply:
intlShim uses `localizeAndFormatMessage` internally 😕

Reply:
Yeah, I saw that, but I think it would be a bit nicer experience if we just exposed a global `intl` object to everyone instead of `localizeMessage`. That way, all of the code using it would look similar and have access to the full functionality. `localizeAndFormatMessage` only exists because `localizeMessage` doesn't accept values and it still doesn't have full formatting support